src/components/Navigator.js

```diff
@@ -0,0 +1,41 @@
+import logoImg from "../assets/svg/Linkbrary.svg";
+import profileImg from "../assets/images/profile.png";
+import "./Navigator.css";
+
+function Profile({ user }) {
+  if (user.email) {
+    return (
+      <div className="navbar-profile">
+        <img
+          className="navbar-profile-img"
+          alt="profile-img"
+          src={profileImg}
+        ></img>
```

Comment:
요런 경우 쌩 html 과 다르게 self-closing 이 가능합니다! 
```suggestion
        />
```